shared/actions/config/index.js

```diff
@@ -2,14 +2,36 @@
 import * as Constants from '../../constants/config'
 import engine from '../../engine'
 
+import {switchTab} from '../../actions/tabbed-router'
+import {navigateTo} from '../../actions/router'
+import {loginTab, moreTab} from '../../constants/tabs'
+
 // $FlowFixMe
 import * as native from './index.native'
 
 import type {AsyncAction} from '../../constants/types/flux'
 import type {config_getConfig_rpc, config_getExtendedStatus_rpc, config_getCurrentStatus_rpc} from '../../constants/types/flow-types'
 
+function switchTabs(dispatch, getState) {
+  const {config: {status}} = getState()
+  if (!status) {
+    return
```

Comment:
Is this an error? we should log if so. In case we get a bug report so we see it in the log